<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PwordEditDlg</class>
 <widget class="QDialog" name="PwordEditDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>351</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Password</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>G&amp;eneral</string>
      </attribute>
      <layout class="QHBoxLayout">
       <item>
        <layout class="QVBoxLayout">
         <item>
          <layout class="QGridLayout">
           <item row="2" column="1">
            <widget class="QLineEdit" name="passwordEdit">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="textLabel1">
             <property name="text">
              <string>&amp;Name:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="buddy">
              <cstring>nameEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="textLabel2">
             <property name="text">
              <string>&amp;User:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="buddy">
              <cstring>userEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="textLabel3">
             <property name="text">
              <string>&amp;Password:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="buddy">
              <cstring>passwordEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="userEdit"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="nameEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QLabel" name="textLabel4">
             <property name="text">
              <string>No&amp;tes:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="buddy">
              <cstring>notesEdit</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="notesEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QPushButton" name="showButton">
           <property name="text">
            <string>&amp;Show</string>
           </property>
           <property name="shortcut">
            <string>Alt+S</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fetchButton">
           <property name="text">
            <string>&amp;Fetch</string>
           </property>
           <property name="shortcut">
            <string>Alt+F</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="genButton">
           <property name="text">
            <string>&amp;Generate</string>
           </property>
           <property name="shortcut">
            <string>Alt+G</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer4_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>119</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>&amp;Details</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="createdOnLabel">
         <property name="text">
          <string>Created on: %1</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="accessedOnLabel">
         <property name="text">
          <string>Accessed on: %1</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="modifiedOnLabel">
         <property name="text">
          <string>Modified on: %1</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lifetimeLabel">
         <property name="text">
          <string>Lifetime: %1</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="uuidLabel">
         <property name="text">
          <string>UUID: %1</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer name="spacer5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>248</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/images/button_cancel.png</normaloff>:/images/button_cancel.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>&amp;Ok</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/images/button_ok.png</normaloff>:/images/button_ok.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+O</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends>Q3Frame</extends>
   <header>q3textedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>userEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>notesEdit</tabstop>
  <tabstop>showButton</tabstop>
  <tabstop>fetchButton</tabstop>
  <tabstop>genButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <includes>
  <include location="global">time.h</include>
 </includes>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>PwordEditDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>PwordEditDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showButton</sender>
   <signal>clicked()</signal>
   <receiver>PwordEditDlg</receiver>
   <slot>togglePassword()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fetchButton</sender>
   <signal>clicked()</signal>
   <receiver>PwordEditDlg</receiver>
   <slot>fetchPassword()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>genButton</sender>
   <signal>clicked()</signal>
   <receiver>PwordEditDlg</receiver>
   <slot>genPassword()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
